ui/app/components/policy-form.hbs

```diff
@@ -60,7 +60,7 @@
               @isIconOnly={{true}}
               @textToCopy={{@model.policy}}
               class="transparent"
-              data-test-copy-button
```

Comment:
upgrading to `3.1.0` removed the test selector `data-test-clipboard` from the copy button - adding the clipboard value to the selector here is the easiest way to address test failures